examples/miral-kiosk/kiosk_main.cpp

```diff
@@ -108,6 +109,12 @@ int main(int argc, char const* argv[])
     MirRunner runner{argc, argv};
 
     DisplayConfiguration display_config{runner};
+ 
+    CommandLineOption show_splash{
+        [&](bool show_splash) {splash(show_splash); },
+        "splash",
+        "Control display splash on startup",
```

Comment:
The option name (and explanation) leaves a little to be desired, I would prefer "show-splash".

Reply:
fixed in pr update